javascript/finishing-up/conclusion.md

```diff
@@ -15,8 +15,8 @@ This section contains helpful links to other content. It isn't required, so cons
 - [An Android development tutorial from CodeLearn](http://www.codelearn.org/android-tutorial)
 - [Giant meta-list of screencasts by Avdi Grimm](https://www.rubytapas.com/2016/06/30/new-list-programming-screencast-series/)
 - [appendTo's Javascript track (incomplete)](http://learn.appendto.com/lesson/javascript-101)
-- [Front End Programming best practices from betterfrontend](http://betterfrontend.com/)
-- [D3 and other visualization tool examples](http://tributary.io)
+- [Front End Programming best practices](https://javascript.plainenglish.io/front-end-web-development-best-practices-from-my-experience-in-the-industry-4c5bbc04c9e3)
```

Comment:
I don't really like the Front End Programming Best Practices link. Since this is in the additional resources section, I would say go ahead and just delete/remove it. 

I think the Visualization Tool Example link is good though